index.html

```diff
@@ -1,51 +1,77 @@
-                <p id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec id nibh sit amet tortor rhoncus pharetra non fermentum quam. Nam dui libero, ultricies a mi vitae, convallis congue felis. Aliquam erat volutpat. In sit amet faucibus tellus, vel semper justo. Fusce ullamcorper pharetra iaculis. Donec venenatis porta dolor, id viverra.</p>
-                <p id="date">24.04.2020</p>
-            </article>
-        </div>
-        <hr>
-    </main>
-    <footer>  
-        <p>Coded by: Wiktor Bonderek</p>
-        <p>wiktor.bonderek@shoper.pl</p>
-    </footer>
+
+  <main>
+
+   <div class="container">
+    
+
+    <div class="header">
+     <div class="back-btn">
+       <span>
+        <i class="fa-solid fa-arrow-left"></i>
+        <a href="#">back</a>
+       </span>
+     </div>
+     <h1>FlexBox App</h1>
+     <div class="settings-btn">
+      <a href="#"><i class="fa-solid fa-gear fa-xs"></i></a>
+
+     </div>
+    </div>
+
+    <div class="text-content">
+     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultrices tempor ultricies. Donec at facilisis dolor. Nulla mollis, ex quis sollicitudin faucibus, lectus diam volutpat erat, a bibendum leo leo vitae odio. In tristique erat ut massa lacinia molestie. Phasellus ut ligula purus. Nam malesuada erat in sollicitudin tristique. Donec sollicitudin at eros ac cursus.</p>
+
+     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultrices tempor ultricies. Donec at facilisis dolor. Nulla mollis, ex quis sollicitudin faucibus, lectus diam volutpat erat, a bibendum leo leo vitae odio. In tristique erat ut massa lacinia molestie. Phasellus ut ligula purus. Nam malesuada erat in sollicitudin tristique. Donec sollicitudin at eros ac cursus.</p>
+
+     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultrices tempor ultricies. Donec at facilisis dolor. Nulla mollis, ex quis sollicitudin faucibus, lectus diam volutpat erat, a bibendum leo leo vitae odio. In tristique erat ut massa lacinia molestie. Phasellus ut ligula purus. Nam malesuada erat in sollicitudin tristique. Donec sollicitudin at eros ac cursus.</p>
+
+     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultrices tempor ultricies. Donec at facilisis dolor. Nulla mollis, ex quis sollicitudin faucibus, lectus diam volutpat erat, a bibendum leo leo vitae odio. In tristique erat ut massa lacinia molestie. Phasellus ut ligula purus. Nam malesuada erat in sollicitudin tristique. Donec sollicitudin at eros ac cursus.</p>
+
+     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultrices tempor ultricies. Donec at facilisis dolor. Nulla mollis, ex quis sollicitudin faucibus, lectus diam volutpat erat, a bibendum leo leo vitae odio. In tristique erat ut massa lacinia molestie. Phasellus ut ligula purus. Nam malesuada erat in sollicitudin tristique. Donec sollicitudin at eros ac cursus.</p>
+
+    </div>
+
+    <div class="nav">
```

Comment:
dlaczego nie uzyłeś tagu `nav` dałeś klase `nav` która sugeruje nawigacje, a jeżeli chodzi o semantyke to nie ma żadnej wartości :)

Reply:
ogólnie to semantyka do poprawy, fromatowanie również, nie wiem czy to github tak tragicznie wyrenderowała czy też tak mas zw edytorze, ale cały dokument jest tragicznie poformatowany, poprzednie zadanie dużo ładniej wygladało

Reply:
prosiłam też zeby nawigacja w htmlu była pierwsza :) a a wyświetlałą sie na dole ale miało to być zrobione przy pomocy cssa